svg-loader.js

```diff
@@ -155,6 +125,9 @@ const renderBody = (elem, options, body) => {
         // Makes sure that class names don't conflict with each other.
         if (elem.tagName === "style" && !disableCssScoping) {
             elem.innerHTML = cssScope(elem.innerHTML, `[data-id="${elemUniqueId}"]`);
```

Comment:
We are assigning `elem.innerHTML` twice. While it doesn't mean a lot since the element hasn't been mounted yet, I think it would be still be better if we have a single variable where it's stored, and assigned in the end.